odata-json-format/7 Structural Property.md

```diff
@@ -409,7 +409,7 @@ OData 4.01 services MUST support both the OData 4.0 representation, for
 requests containing an `OData-Version` header with a value of
 `4.0`, and the OData 4.01 representation, for requests
 containing an `OData-Version` header with a value of `4.01`.
-Clients MUST NOT use `@odata.bind` in requests with an
+Clients MUST NOT use `@bind` in requests with an
```

Comment:
In 4.0 services clients MUST use `@odata.bind`, and in 4.01 or later they MUST NOT use it at all. Please revert this change.